<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnmain">
         <property name="text">
          <string>主页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btntask">
         <property name="text">
          <string>学习任务</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btngoal">
         <property name="text">
          <string>打卡任务</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="helloLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>你好，                       </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnperson">
         <property name="text">
          <string>个人中心</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page1">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>1231</width>
          <height>791</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#000000;&quot;&gt;学习习惯养成计划&lt;/span&gt;&lt;span style=&quot; font-size:24pt; color:#aa00ff;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#000000;&quot;&gt;简介&lt;/span&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;     使用纸质档的记录很容易遗忘自己设定了哪些任务以及复习时间，且没有提醒功能，这样就会让计划的任务落空，任务滚雪球式的累积会让记忆压力越来越大。 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;    软件的用户可以自己添加学习任务，每个学习任务都有自己的主题、完成时间和优先级，根据设定的完成时间和优先级对用户进行不同频率的提醒，直到其完成该任务。 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;    还会给每个任务设定其属性，如需要复习记忆的任务，设定成“需提醒任务”，那么在其完成后的规定时间内会弹出复习提醒，这样可以提醒用户及时复习回顾知识，做到更好的记忆效果。 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;    软件还提供任务分析功能，针对用户每天的任务添加数和完成率进行统计，来让使用者更好的知道自己的复习进度和完成率。通过可视化的图表展示数据，使用者能够有完成任务的成就感。 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page2">
       <property name="styleSheet">
        <string notr="true">#page2{
	
}</string>
       </property>
       <widget class="QToolBox" name="tbUnready">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>351</width>
          <height>701</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>351</width>
           <height>675</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 1</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>1070</x>
          <y>20</y>
          <width>321</width>
          <height>441</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTextBrowser" name="firstTb">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTextBrowser" name="secondTb">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="btnauto">
        <property name="geometry">
         <rect>
          <x>1140</x>
          <y>610</y>
          <width>251</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>任务管理</string>
        </property>
       </widget>
       <widget class="QToolBox" name="tbReady">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>50</y>
          <width>351</width>
          <height>701</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>351</width>
           <height>675</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 1</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>0</y>
          <width>361</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>未完成任务</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>0</y>
          <width>431</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>已完成任务（仅显示7天内）</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page3">
       <property name="styleSheet">
        <string notr="true">#page3{

}</string>
       </property>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>601</width>
          <height>441</height>
         </rect>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTextBrowser" name="dateLog">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>110</y>
          <width>601</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>461</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 18pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>打卡日历</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>30</y>
          <width>461</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 18pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>打卡日志</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page4">
       <property name="styleSheet">
        <string notr="true">#page4{
	
}</string>
       </property>
       <widget class="QPushButton" name="btnoutlogin">
        <property name="geometry">
         <rect>
          <x>1150</x>
          <y>10</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>退出登录</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>671</width>
          <height>271</height>
         </rect>
        </property>
        <property name="text">
         <string>通过可视化的图表展示数据，使用者能够有完成任务的成就感。</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
